source/auth_n.rst

```diff
@@ -4915,7 +4869,7 @@ Challenging During Factor Creation
 
```

Comment:
Below the phrase `Once you have collected the code from the user, send the code generated by your Google Authenticator app:` there is Java Code missing